lib/modules/versioning/unity3d/readme.md

```diff
@@ -0,0 +1,5 @@
+Unity versioning follow semantic versioning, followed by a letter, number and hash.
```

Comment:
Please add one or two examples, so it is easier to understand what the version looks like. 

People searching versioning for example for usage by customManagers.

Reply:
Extended the readme in 077827bc5500f767004a1c7536b54774fc478b10.